metadata_tools/cumulative_support.py

```
@@ -125,44 +126,45 @@ def create_cumulative_indexes(template_name, exclude=None):
         exclude (list, optional): List of volumes to exclude.
     """
     # Parse arguments
-    host, index_type = util.parse_template_name(template_name)
+    host, index_type, template_dir = util.parse_template_name(template_name)
```

Ruff (0.14.10)

Unpacked variable `index_type` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)